### PR TITLE
Products table

### LINKED ISSUE #43: Add Style to Product Table

### PR DESCRIPTION
**Pull request by Arique1104**
This PR:
- Adds a new migration to add a `style` column to product table.
- Updates seeds so products also have a style column

Closes #43

### CODE REVIEW

File: code/api/src/migrations/20201207225501-modify_products_add_style.js [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+        ),
+      ]);
+    },
+    down(queryInterface, Sequelize) {
+      // logic for reverting the changes
+      return Promise.all([
+        queryInterface.removeColumn('products', 'style'),
+      ]);
+    },
+  };
```
Comment (LHJE) [inline code/api/src/migrations/20201207225501-modify_products_add_style.js:22]: So glad we were able to make this work!  Great job!

**Review by LHJE — APPROVED** [review]
Great job team!

File: code/api/src/config/config.json [hunk 2]
```diff
@@ -0,0 +1,16 @@
...
+    "dialect": "postgres"
+  },
+  "production": {
+    "username": "postgres",
+    "password": null,
+    "database": "crate",
+    "host": "127.0.0.1",
+    "dialect": "postgres"
+  }
+}
```
Comment (drewwilliams5280) [inline code/api/src/config/config.json:16]: Still not sure why this was created and why we can't use the db.json. Will research.

File: code/api/src/seeders/2-products.js [hunk 3]
```diff
@@ -10,6 +10,7 @@ module.exports = {
         slug: 'belt-for-women',
         description: 'A very nice belt for women.',
         type: params.product.types.accessory.id,
+        style: "Classy",
         gender: params.user.gender.female.id,
```
Comment (drewwilliams5280) [inline code/api/src/seeders/2-products.js:14]: We're not going to forget this, but just for the sake of documentation, we need more seeds so we can have a lot of data, and we also need more style types.

**Pull request merged**